packages/builder/src/components/common/CodeEditor/CodeEditor.svelte

```diff
@@ -71,7 +73,7 @@
 
   $: {
     if (autofocus && isEditorInitialised) {
-      editor.focus()
+      editor?.focus()
```

Comment:
This should not be required if you remove the ` | undefined` from `let textarea: HTMLDivElement`